velox/functions/lib/DateTimeFormatter.cpp

```diff
@@ -735,7 +740,6 @@ int32_t parseFromPattern(
       return -1;
     }
     cur += size;
-    date.weekDateFormat = true;
```

Comment:
I wonder why we need this change, and could you clarify what is covered by `weekDateFormat` which seems to be a larger scope than `weekOfMonthDateFormat`?